tests/unit/test_storage.py

```diff
@@ -336,8 +336,37 @@ def test_superblock_read_error(self, mock_issue_utils):
         inst()
         self.assertTrue(mock_issue_utils.add_issue.called)
 
+    @mock.patch.object(ceph_cluster_checks, 'issue_utils')
+    def test_check_crushmap_equal_buckets(self, mock_issue_utils):
+        inst = ceph_cluster_checks.CephClusterChecks()
+        inst.check_crushmap_equal_buckets()
+        self.assertFalse(mock_issue_utils.add_issue.called)
+
+    @mock.patch.object(ceph_cluster_checks, 'issue_utils')
+    def test_check_crushmap_non_equal_buckets(self, mock_issue_utils):
+        '''
+        Verifies that the check_crushmap_equal_buckets() function
+        correctly raises an issue against a known bad CRUSH map.
+        '''
+        test_data_path = ("sos_commands/ceph/json_output/"
+                          "ceph_osd_dump_--format_json-pretty_unbalanced")
```

Comment:
the above path should be ceph_osd_crush_dump_ ... (not ceph_osd_dump). not a biggie, we can fix after this lands.